app_changelog.go

```diff
@@ -0,0 +1,5 @@
+package versionbundle
+
+type AppsChangelogs map[string][]Changelog
+
+type ReleasesChangelogs map[string]AppsChangelogs
```

Comment:
Sorry to be late here but I don't get the plural here. It should be `AppChangelog` IMO. It's a changelog for a single app correct?